docs/src/osdk/guide/run-project.md

```diff
@@ -76,12 +76,12 @@ cargo osdk test foo
 
 ## Options
 
-Both `build`, `run`, and `test` commands accept options
+All of the `build`, `run`, and `test` commands accept options
 to control their behavior, such as how to compile and
 launch the kernel.
 The following documentations provide details on
 all the available options:
 
-- [build options](../reference/commands/build.md)
-- [run options](../reference/commands/run.md)
-- [test options](../reference/commands/test.md)
+- [`build` options](../reference/commands/build.md)
+- [`run` options](../reference/commands/run.md)
+- [`test` options](../reference/commands/test.md)
```

Comment:
I'm not sure how this would look like when formatted into the book, any advice?

Reply:
I think the changes will be formatted just fine. If you want to preview the rendered book, run these commands

```
cd docs
mdbook serve --open
```